src/runner.rs

```diff
@@ -224,4 +228,37 @@ When your transaction is complete, enter 'commit' or 'abort' as appropriate."#,
         }
         Ok(())
     }
+
+    pub(crate) async fn handle_status(&self) -> Result<()> {
+        // TODO: Return latency information from recent commands if we're able to capture it.
+        self.deps.ui.println(&format!("Current region: {}", self.deps.env.region().value.name()));
```

Comment:
Seems a little verbose, perhaps just `$REGION - $LEDGER - $SHELL_VERSION`?

Reply:
Updated, now it will return:

```
Region: us-east-1, Ledger: vehicle-registration, Shell version: 2.0.0-alpha11
```